x11-wm/marco/marco-1.28.1-r1.ebuild

```diff
@@ -60,6 +63,12 @@ BDEPEND="${COMMON_DEPEND}
 	xinerama? ( x11-base/xorg-proto )
 "
 
+src_prepare() {
+	default
+	# Bug 906826, tracker bug 762406
+	python_fix_shebang meson_post_install.py
```

Comment:
This is unnecessary, since the script uses `#!/usr/bin/env python3` and it will detect the wrapper created by `python_setup` / `python-any-r1_pkg_setup`

Reply:
Okay, this is updated, not sure if I understood correctly so don’t hesitate to tell me if it needs another fix.